File: 30-days-of-code/cpp/Day3.cpp
#include <iostream>
#include <string>

using namespace std;

string conditionalStatement(int i)
{
    // return conditional statement
    if (i % 2 != 0)
        return "Weird";
    if (i > 20)
        return "Not Weird";
    if (i >= 6)
        return "Weird";

    return "Not Weird";
}

int main()
{
    // read input
    string s;
    getline(cin, s);
    int i = stoi(s);

    // print conditional statement
    cout << conditionalStatement(i);
    return 0;
}

File: 30-days-of-code/java/Day3.java
import java.util.*;

class Day3 {
    static String conditionalStatement(int i) {
        // return conditional statement
        if (i % 2 != 0)
            return "Weird";
        if (i > 20)
            return "Not Weird";
        if (i >= 6)
            return "Weird";

        return "Not Weird";
    }

    public static void main(String[] args) {
        // create scanner
        Scanner in = new Scanner(System.in);

        // read input
        int i = in.nextInt();

        // close scanner
        in.close();

        // print conditional statement
        System.out.println(conditionalStatement(i));
    }
}
